### DISCUSSION TITLE
Album organizing

### DISCUSSION THREAD

**Discussion by shawn1thompson** [ente-io/ente#527] [Enhancements]
Is there a way to create new albums? The only way I've seen is when uploading, but that didn't work for me even though I selected to create a new album.

**Comment by vishnukvmd** [comment]
Hey! Sorry, currently there's no way to create empty albums :(
This is on [our roadmap](https://roadmap.ente.io/option-to-create-an-empty-album-first-p-4561/) and we will fix it soon. In the meanwhile, as a workaround, you could select uploaded files and add/move them to a new album.
Will update this issue once we've implemented the desired interaction.

**Reply by spaasis** [reply]
The roadmap link seems to be broken - is there any plans on this feature? I was very confused when I was attempting to create new albums. Found it eventually, but a more direct route would be appreciated

**Reply by crKRoEYS** [reply]
It's possible: Ente Photos App -> Switch to "On Ente" -> to the right is a "+" symbol

**Reply by spaasis** [reply]
Sorry, I meant the web ui. Works as expected in the app
